services/src/main/java/org/keycloak/authentication/authenticators/client/AbstractJWTClientValidator.java

```diff
@@ -117,13 +117,12 @@ private boolean validateClient() {
             return false;
         }
 
-        context.getEvent().client(clientId);
```

Comment:
I am not 100% confident about this change as if request fails due the client not found, it may be useful for administrator to see the `client_id` in the event details to know what was the client-id attempted in the request?